health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -0,0 +1,520 @@
+            "code": "MICROPLAN_MO_01_",
+            "name": "MICROPLAN_MO_01_",
+            "type": "Province",
+            "parent": "MICROPLAN_MO",
+            "isRoot": false,
+            "includeAllChildren": false
+        }
+    ] // Memoized to prevent recomputation on every render
+
+    // Memoizing parents and parent_group to only compute once when selectedData changes
+    const { parents, parent_group } = useMemo(() => {
+        let parents = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'parents' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)